entrypoint.sh

```diff
@@ -80,3 +80,4 @@ helm lint .
 helm package . ${REGISTRY_APPVERSION} ${REGISTRY_VERSION} ${UPDATE_DEPENDENCIES}
 helm inspect chart *.tgz
 helm cm-push *.tgz ${REGISTRY_URL} ${REGISTRY_USERNAME} ${REGISTRY_PASSWORD} ${REGISTRY_ACCESS_TOKEN} ${FORCE}
+rm -rf ${CHART_FOLDER}/charts/*.tgz ${CHART_FOLDER}/*.tgz *.tgz
```

Comment:
This command can fail if there is a lot of *.tgz files, because bash actually expands the asterisk to every matching file, producing a very long command line.
Consider using something like
>find . -name '*.tgz' -delete